app/models/miq_request_workflow.rb

```diff
@@ -742,11 +727,9 @@ def set_default_user_info
 
     if get_value(@values[:owner_email]).blank? && @requester.email.present?
       @values[:owner_email] = @requester.email
-      retrieve_ldap if MiqLdap.using_ldap?
     end
 
-    show_flag = MiqLdap.using_ldap? ? :show : :hide
-    show_fields(show_flag, [:owner_load_ldap])
+    show_fields(:hide, [:owner_load_ldap])
```

Comment:
I don't understand what this does now, so can it be removed entirely?

Reply:
I see it in dialogs, such as manageiq-providers-azure/content/miq_dialogs/miq_provision_azure_dialogs_template.yaml:75